.github/workflows/build.yaml

```
@@ -97,7 +97,7 @@ jobs:
             type=sha,format=long,prefix=
 
       - name: Docker Build
-        uses: docker/build-push-action@v5
+        uses: docker/build-push-action@v6
```

mind reverting this line specifically? I'd like to keep the `build-docker` stage the same as the existing beeper one for consistency where possible

```suggestion
        uses: docker/build-push-action@v5
```

---

Sure!